src/package/nuspec/TestPlatform.Build.nuspec

```diff
@@ -14,7 +14,7 @@
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
   </metadata>
   <files>
-    <file src="Microsoft.TestPlatform.Build\netstandard1.3\Microsoft.TestPlatform.Build.dll" target="lib\netstandard1.3\" />
-    <file src="Microsoft.TestPlatform.Build\netstandard1.3\Microsoft.TestPlatform.targets" target="runtimes\any\native" />
+    <file src="Microsoft.TestPlatform.Build\Debug\netstandard1.3\Microsoft.TestPlatform.Build.dll" target="lib\netstandard1.3\" />
```

Comment:
The packaging may not depend on configuration, the nuspec will always expect Debug bits. In official CI, we build release configuration.

> I might be confused about https://github.com/Microsoft/vstest/pull/552/files#diff-56d541dcd7f193797f0c7bc290c82117L354, are the files not supposed to go in separate folders based on framework ( a netstandard1.3 one and a net46 one)

You're right, they should go in separate folders. Today only a directory is created for `netstandard1.3` is because of the typo in `$binariesDirectory = [System.IO.Path]::Combine("src", "$packageName", "bin", "$TPB_Configuration", "*.*")`; the pattern should be `*`, not `*.*`.